src/services/ApiService.ts

```diff
@@ -44,6 +44,25 @@ class ApiService {
 			})
 	}
 
+	/**
+	 * Use the fetch API over HTTP to hit user search endpoint on the server
+	 *
+	 * @returns Promise that resolves to user object if successful; otherwise, Promise that resolves to empty object
+	 */
+	 pingUserSearchEndpoint(searchKey: string): Promise<any> {
+		return fetch(`${this._apiUrl}/user/search?searchKey=${searchKey}`, {
+			cache: NO_CACHE,
+			method: HTTP_GET,
+		})
```

Comment:
I feel like this should be formatted differently, but if you ran Prettier then you can ignore this suggestion

```suggestion
		return fetch(
			`${this._apiUrl}/user/search?searchKey=${searchKey}`,
			{
				cache: NO_CACHE,
				method: HTTP_GET,
			})
```